include/lsst/meas/base/ApertureFlux.h

```diff
@@ -234,9 +234,9 @@ struct ApertureFluxResult : public FluxResult {
 };
 
 /**
- *  Measurement transformation for aperture fluxes
+ *  Measurement transformation for aperture instFluxes
  *
- *  Transforms fluxes with associated errors to magnitudes. Correctly handles
+ *  Transforms instFluxes with associated errors to magnitudes. Correctly handles
```

Comment:
Does this truly only change instFluxes or both kinds of fluxes?

Reply:
Looking at the code, it is an instFlux->magnitude transform only. Though I'm a little curious why it even exists.